src/urllib3/connection.py

```diff
@@ -369,6 +372,19 @@ def request_chunked(
         """
         self.request(method, url, body=body, headers=headers, chunked=True)
 
+    def getresponse(self, **response_kw: Any):
+
+        # Get the response from http.client.HTTPConnection
+        httplib_response = super().getresponse()
+        from .response import HTTPResponse
```

Comment:
This was the only way I could get this to work without circular import

I'm going to put it at the top of the funciton with a comment to explain why its here.